qa-core/src/config/internal-api/TestConfiguration/rows.ts

```diff
@@ -0,0 +1,28 @@
+import { Response } from "node-fetch"
+import { Row } from "@budibase/types"
+import InternalAPIClient from "./InternalAPIClient"
+
+export default class RowsApi {
+    api: InternalAPIClient
+
+    constructor(apiClient: InternalAPIClient) {
+        this.api = apiClient
+    }
+
+    async getAll(id: string): Promise<[Response, Row[]]> {
```

Comment:
Might make sense to make this explicit with `tableId`

Reply:
Artifact from when this method was part of tables class. Thank you for pointing that out